webapp/app/templates/lotse/form_steuernummer.html

```diff
@@ -0,0 +1,185 @@
+{% extends 'basis/form_standard.html' %}
+{% import "macros.html" as macros %}
+
+
+{% block form_content %}
+    <div class="form-row-center">
+        {{ macros.render_field(form.steuernummer_exists, first_field=True if not render_info.form.errors) }}
+    </div>
+    <div class="form-row-center">
+        {{ macros.render_field(form.bundesland) }}
+    </div>
+
+    <div class="form-row-center">
+        {{ macros.render_field(form.bufa_nr) }}
+    </div>
+
+    <div class="form-row-center">
+        {{ macros.render_field(form.steuernummer) }}
+    </div>
+
+    <div class="additional-input-content" for="request_new_tax_number">
+        <h2 class="form-sub-heading-smaller">{{ _('form.lotse.request_new_tax_number.title') }}</h2>
+        {{ _('form.lotse.request_new_tax_number.intro') }}
+        {{ macros.render_field(form.request_new_tax_number) }}
+    </div>
+
+{% endblock %}
+
+{% block optional_js %}
+    {{ super() }}
+    <script src="{{ url_for('static', filename='js/field_utils.js') }}" type=text/javascript></script>
+    <script>
+        let taxOffices = JSON.parse('{{ render_info.additional_info['tax_offices']|tojson }}');
```

Comment:
The value of `taxOffices` does not change, so it should be `const` not `let`. Applies to the rest of the variable declarations in this file. Basically, variable declarations should be `const` _unless_ you need to write `varname = newValue` anywhere.